docs/en/latest/practices/how-to-access-Apache-APISIX-Prometheus-Metrics-on-k8s.md

```diff
@@ -0,0 +1,97 @@
+
+## Initial Knowledge about `public-api`
+
+When users develop custom plugins in Apache APISIX, they can define some APIs (referred to as public API) for the plugins. The provided interface is for internal calls in practical application scenarios rather than being open on the public network for anyone to call.
+
+Therefore, Apache APISIX has designed a `public-api` plugin. With this plugin, you can solve the public API's pain points. You can set a custom URI for the public API and configure any plugin. For more information, see the `public-api` Plugin's official [document](https://apisix.apache.org/docs/apisix/plugins/public-api/).
+
+The primary role of the `public-api` plugin in this document is to protect the URI exposed by *Prometheus*.
+
+**Note**: We should note that this feature is only available in APISIX version 2.13 and later.
+
+## Begin to access Apache APISIX Prometheus Metrics
+
+Before starting, please make sure that Apache APISIX (version >= 2.13)and APISIX Ingress controller are installed and working correctly.
+
+### Step 1: Enable Prometheus Plugin
+
+If you need to monitor Apache APISIX simultaneously, you can create the following ApisixClusterConfig resource.
+
+```yaml
+apiVersion: apisix.apache.org/v2beta3
+kind: ApisixClusterConfig
+metadata:
+  name: default
+spec:
+  monitoring:
+    prometheus:
+      enable: true
+```
+
+### Step 2: Enable `public-api` Plugin
+
+Let's make a basic routing setup, and please note that further configuration should be done based on your local backend service information. The primary solution concept is to use the `public-api` plugin to protect the routes exposed by *Prometheus*. For a more detailed configuration, you can refer to the [example](https://apisix.apache.org/zh/docs/apisix/plugins/public-api/#example) section of the `public-api` plugin.
+
+```yaml
+apiVersion: apisix.apache.org/v2beta3
+kind: ApisixRoute
+metadata:
+  name: prometheus-route
+spec:
+  http:
+  - name: public-api
+    match:
+      hosts:
+      - test.prometheus.org
+      paths:
+      - /apisix/prometheus/metrics
+    backends:
+    ## Please notice that there must be your actual "serviceName" and "servicePort"
+    - serviceName: apisix-test-prometheus
+      servicePort: 9180
+    plugins:
+    - name: public-api
+      enable: true
+```
+
+### Step 3: Collect the Metrics
+
+Now you can then get the indicator parameters by requesting command access.
+
+```sh
+kubectl exec -it -n ${namespace of Apache APISIX} ${Pod name of Apache APISIX} -- curl http://127.0.0.1:9180/apisix/admin/routes -H 'X-API-Key: edd1c9f034335f136f87ad84b625c8f1'
+
+kubectl exec -it -n ${namespace of Apache APISIX} ${Pod name of Apache APISIX} -- curl http://127.0.0.1:9080/headers -H 'Host: test.prometheus.org'
```

Comment:
You can show the output contents. 

This way we know what the expected result should be